project/ScalafixBuild.scala

```diff
@@ -56,11 +56,10 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
           }
 
           val prevVersions = previousVersions(sv).map(prev => TargetAxis(prev))
-          val scala3FromScala2 = TargetAxis(scala3)
```

Comment:
that seemed a bit unnecessary/overkill, so I specialized it to reduce build complexity and test time
```diff
 sbt:scalafix> projects
 [info] In file:/home/brice/git/scala/scalafix/
 [info] 	   cli2_12
 [info] 	   cli2_13
 [info] 	   cli3
 [info] 	   core2_12
 [info] 	   core2_13
 [info] 	   core3
 [info] 	   docs2_13
 [info] 	   expect2_12Target2_12_18
 [info] 	   expect2_12Target2_12_19
 [info] 	   expect2_12Target2_12_19xsource3
-[info] 	   expect2_12Target3_4_1
 [info] 	   expect2_13Target2_13_12
 [info] 	   expect2_13Target2_13_13
 [info] 	   expect2_13Target2_13_14
 [info] 	   expect2_13Target2_13_14xsource3
 [info] 	   expect2_13Target3_4_1
 [info] 	   expect3Target3_4_1
 [info] 	   input2_12_18
 [info] 	   input2_12_19
 [info] 	   input2_12_19xsource3
 [info] 	   input2_13_12
 [info] 	   input2_13_13
 [info] 	   input2_13_14
 [info] 	   input2_13_14xsource3
 [info] 	   input3_4_1
 [info] 	   integration2_12
 [info] 	   integration2_13
 [info] 	   integration3
 [info] 	   interfaces
 [info] 	   output2_12_19
 [info] 	   output2_12_19xsource3
 [info] 	   output2_13_14
 [info] 	   output2_13_14xsource3
 [info] 	   output3_4_1
 [info] 	   reflect2_12
 [info] 	   reflect2_13
 [info] 	   reflect3
 [info] 	   rules2_12
 [info] 	   rules2_13
 [info] 	   rules3
 [info] 	 * scalafix
 [info] 	   shared2_12
 [info] 	   shared2_13
 [info] 	   shared3
 [info] 	   testkit2_12
 [info] 	   testkit2_13
 [info] 	   testkit3
 [info] 	   unit2_12
 [info] 	   unit2_13
 [info] 	   unit3
```